cluster-autoscaler/cloudprovider/azure/README.md

```diff
@@ -120,4 +122,65 @@ To run a CA pod with Azure managed service identity (MSI), use [cluster-autoscal
 
 ```sh
 kubectl create -f cluster-autoscaler-standard-msi.yaml
-```
\ No newline at end of file
+```
+
+### AKS or ACS deployment
+
+Pre-requirements:
+
+- Get credentials from above `permissions` step.
+- Get the cluster name using the following:
+
+  for AKS:
+  ```sh
+  az aks list
+  ```
+  for ACS:
+  ```sh
+  az acs list
+  ```
+
+- Get a node pool name by extracting the value of the label **agentpool**
+  ```sh
+  kubectl get nodes --show-labels
+  ```
+- In case of AKS we need additional information in the form of node resource group.
+  Use the value of the label by name **kubernetes.azure.com/cluster** as the node resource group.
+
+- Encode each data with base64.
+
+Fill the values of cluster-autoscaler-azure secret in [cluster-autoscaler-containerservice](cluster-autoscaler-containerservice.yaml), including
+
+- ClientID: `<base64-encoded-client-id>`
```

Comment:
we need to document somewhere that this service principal needs write access to the cluster resource. That will often be the case but not always and it would be very frustrating in the cases it doesn't have access.

Reply:
The Permissions section above, explicitly states the role as Contributor. Would that suffice ?

Reply:
yes that will work, didn't see that.